python/lsst/dax/imgserv/vo/imageSODA.py

```diff
@@ -49,11 +49,15 @@ class ImageSODA(SODA):
     """ Class to handle SODA operations.
 
     For reference, LSST camera filters (Filter, Blue Side, Red Side):
-        G : 400 - 552
+        U : 324 = 305
+        G : 405 - 552
         R : 552 - 691
         I : 691 - 818
-        Z : 818 - 922
-        Y : 948 - 1060
+        Z : 818 - 921
+        Y : 922 - 997
+    (Ref: https://lsst.org/scientists/keynumbers)
+
+    In SODA, these correspond to possible values for the BAND parameter.
```

Comment:
It may be better to indicate these values are set from BAND in python/lsst/dax/imgserv/config/image_api_schema.json so that only one set needs to be maintained. If you keep them, change the '=' in U to '-'

Reply:
Typo fixed. Will keep this as the new comment added in image_api_schema.json won't show up as docstrings.